c++/src/kj/compat/tls.h

```diff
@@ -97,6 +97,9 @@ class TlsContext: public kj::SecureNetworkWrapper {
     //   algorithms.
     // - You need quickly to disable an algorithm recently discovered to be broken.
 
+    kj::Maybe<kj::StringPtr> curveList;
+    // Sets the preferred curves (Groups in TLS 1.3), by default this is not set.
```

Comment:
I've not got the domain knowledge to know whether this is self-explanatory, but is it clear what format each "curve" should be in when specified as a string? The [docs](https://www.openssl.org/docs/man1.0.2/man3/SSL_CTX_set1_curves_list.html) talk about "curve NIDs".

Reply:
Yes, colon separated just like the cipher list.

> [Each EC_builtin_curve item has a unique integer id (nid), and a human readable comment string describing the curve.](https://www.openssl.org/docs/manmaster/man3/EC_GROUP_new.html)

Reply:
What I'm saying is that this should be documented :P

Reply:
I linked to the source of the NID's and names in boring ssl.